linux-build.sh

```diff
@@ -2,33 +2,7 @@
 
 set -euxo pipefail
 
-ARCH=""
-
-POSITIONAL=()
-while [[ $# -gt 0 ]]
-do
-key="$1"
-
-case $key in
-    --GOARCH)
-    ARCH="$2"
-    shift # past argument
-    shift # past value
-    ;;
-    --default)
-    DEFAULT=YES
-    shift # past argument
-    ;;
-    *)    # unknown option
-    POSITIONAL+=("$1") # save it in an array for later
-    shift # past argument
-    ;;
-esac
-done
-set -- "${POSITIONAL[@]}" # restore positional parameters
-
-
 VERSION_TAG="$(git rev-parse HEAD)"
 VERSION_LINK_FLAG="main.gitCommit=${VERSION_TAG}"
 
-CGO_ENABLED=0 GOOS=linux GOARCH=$ARCH go build -a -ldflags "-extldflags '-static' -X ${VERSION_LINK_FLAG}" .
+CGO_ENABLED=0 GOOS=linux GOARCH=$GOARCH go build -a -ldflags "-extldflags '-static' -X ${VERSION_LINK_FLAG}" .
```

Comment:
`GOARCH=$GOARCH` isn't needed, since it's already an environment variable.

Reply:
I know, but you get a nice output when building. I considered using `echo $GOARCH` but I like this style better, since it's consistent across. What do you think?

Reply:
I think this would break if `GOARCH` isn't set in the environment, since we have `set -u` in the script. So maybe change this to `GOARCH=${GOARCH:-amd64}` ?

Reply:
It's set in the  `Dockerfile`: `ARG GOARCH="amd64"`. I would not want to duplicate it in the `.sh` as well. What do you think?

Reply:
It's handy to be able to run linux-build.sh without docker, let's not break that.

Reply:
Oh yes, true. I've added this option. :+1: